backend/app.go

```diff
@@ -70,3 +74,8 @@ func (a *App) StartBuild() {
 		runtime.EventsEmit(a.ctx, "build:log", fmt.Sprintf("Error: build failed: %v", err))
 	}
 }
+
+// StartGitHubLogin starts the GitHub device flow authentication
+func (a *App) StartGitHubLogin() (*auth.UserCodeInfo, error) {
```

Comment:
bskit checks this stunt for methods that need to get bindings for the FE can use